mmv1/templates/terraform/encoders/monitoring_service_generic.go.erb

```diff
@@ -0,0 +1,27 @@
+<%# The license inside this block applies to this file.
+	# Copyright 2022 Google Inc.
+	# Licensed under the Apache License, Version 2.0 (the "License");
+	# you may not use this file except in compliance with the License.
+	# You may obtain a copy of the License at
+	#
+	#     http://www.apache.org/licenses/LICENSE-2.0
+	#
+	# Unless required by applicable law or agreed to in writing, software
+	# distributed under the License is distributed on an "AS IS" BASIS,
+	# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	# See the License for the specific language governing permissions and
+	# limitations under the License.
+-%>
+// query only parameters below
```

Comment:
Why do we remove all of these? They aren't present in the TF schema, so why would they ever exist in the request?

Reply:
They don't exist in the request, but they do appear in the response. I removed them to avoid having Terraform see them in the response and incorrectly assessing that the state-of-the-world (as expressed by the responses) have differences in the plan.

Reply:
Encoders run on the request object prior to it being sent to the API, so this isn't having the intended effect. 

See https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-6604-old..auto-pr-6604#diff-9dac932a2275186a608e96784d2b652b92cfc234cb0451c12c0ad45853b3e855R162

Reply:
Which indicates that it isn't needed on the response side either.

Reply:
Odd. I was getting diffs in my `terraform plan` that went away when I added this encoder. But it looks like you're right - this is unnecessary.